plugins/scaffolder-backend/src/scaffolder/actions/builtin/publish/github.ts

```diff
@@ -128,6 +130,7 @@ export function createPublishGithubAction(options: {
         access,
         requireCodeOwnerReviews = false,
         requiredStatusCheckContexts = [],
+        requireBranchesToBeUpToDate = true,
```

Comment:
Does this default to true or false without it? I feel that it might be false, in which case we should default to false. Don't want to change and break existing behaviour without an opt in 😄

Reply:
I'll create a new repo tomorrow and call the API directly and see what value it ends up with. I _think_ it defaults to true on the GitHub side anyway, but I'll check 👍

Reply:
I just created a brand new public GitHub repo and as soon as I ticked the _Require status checks to pass before merging_ checkbox for a branch protection rule, it rendered the UI to configure the status checks with the _Require branches to be up to date before merging_ setting checked.

![image](https://user-images.githubusercontent.com/1439341/194260971-da720879-28c8-440c-9ad3-da2f121efdf5.png)